deploy/helm/templates/metabase-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.metabase.enabled }}
+{{- $fullName := include "app.fullname" . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-metabase
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $fullName }}-metabase
+  template:
+    metadata:
+      labels:
+        app: {{ $fullName }}-metabase
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: "cd-serviceaccount"
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: metabase
+          image: metabase/metabase:v0.48.8
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: 3000
+          resources:
+            requests:
+              cpu: 25m
+              memory: 1Gi
+            limits:
+              cpu: 500m
+              memory: 3Gi
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsUser: 1000
+            runAsGroup: 1000
+          env:
+            - name: MB_DB_CONNECTION_URI
+              valueFrom:
+                secretKeyRef:
+                  name: rds-postgresql-read-replica-output
```

Comment:
We don't have a read-replica in cfe-civil. I think this needs to be `rds-postgresql-instance-output` (no need to create a read-replica tbh, since our database is not used much)

Reply:
cfe-civil does have a read_replica, I think they get created by default when using the cli to create an rds resource.
https://github.com/ministryofjustice/cloud-platform-environments/blob/ba6bde946647a3dae3bd40857f3317473f08ad53/namespaces/live.cloud-platform.service.justice.gov.uk/cfe-civil-production/resources/rds.tf#L62
I am happy to us the main db

Reply:
ah, good spot, however it's not created in our case because `count  = 0` https://github.com/ministryofjustice/cloud-platform-environments/blob/ba6bde946647a3dae3bd40857f3317473f08ad53/namespaces/live.cloud-platform.service.justice.gov.uk/cfe-civil-production/resources/rds.tf#L64 hence why the `rds-postgresql-read-replica-output` secret doesn't exist